cli/src/lib.rs

```diff
@@ -132,11 +133,12 @@ impl NodeBuilder {
                 .with_factory(Supervisor)
                 .with_factory(ExplorerFactory),
             external_runtimes: vec![],
+            default_instances: vec![],
```

Comment:
I don't think `default_instances` is a good name; we call the services started in the genesis block *built-in* services. `DefaultInstance` does not necessarily mean that its parameters will be used in the genesis block.